app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenter.kt

```diff
@@ -93,44 +95,63 @@ class BrowserToolbarCFRPresenter(
     @VisibleForTesting
     internal fun showTcpCfr() {
         CFRPopup(
-            text = context.getString(R.string.tcp_cfr_message),
             anchor = toolbar.findViewById(
                 R.id.mozac_browser_toolbar_security_indicator,
             ),
             properties = CFRPopupProperties(
                 popupAlignment = INDICATOR_CENTERED_IN_ANCHOR,
+                popupBodyColors = listOf(
+                    getColor(context, R.color.fx_mobile_layer_color_gradient_end),
+                    getColor(context, R.color.fx_mobile_layer_color_gradient_start),
+                ),
+                popupVerticalOffset = CFR_TO_ANCHOR_VERTICAL_PADDING.dp,
+                dismissButtonColor = getColor(context, R.color.fx_mobile_icon_color_oncolor),
                 indicatorDirection = if (settings.toolbarPosition == ToolbarPosition.TOP) {
                     CFRPopup.IndicatorDirection.UP
                 } else {
                     CFRPopup.IndicatorDirection.DOWN
                 },
-                popupVerticalOffset = CFR_TO_ANCHOR_VERTICAL_PADDING.dp,
             ),
             onDismiss = {
                 when (it) {
                     true -> TrackingProtection.tcpCfrExplicitDismissal.record(NoExtras())
                     false -> TrackingProtection.tcpCfrImplicitDismissal.record(NoExtras())
                 }
             },
-        ) {
-            Text(
-                text = context.getString(R.string.tcp_cfr_learn_more),
-                color = FirefoxTheme.colors.textOnColorPrimary,
-                modifier = Modifier.clickable {
-                    context.components.useCases.tabsUseCases.selectOrAddTab.invoke(
-                        SupportUtils.getSumoURLForTopic(
-                            context,
-                            TOTAL_COOKIE_PROTECTION,
+            text = {
+                FirefoxTheme {
+                    Text(
+                        text = context.getString(R.string.tcp_cfr_message),
+                        modifier = Modifier.padding(
+                            end = 24.dp, // 8.dp extra padding to the "X" icon
                         ),
```

Comment:
Here and below: This padding won't be needed. The AC CFR will add this padding to ensure proper positioning (and the result will be the same UI Focus/Fenix had before).